### ISSUE TITLE
How do I create a data zoo

### ISSUE THREAD

**Issue by 6sephiruth** [ZhaoJ9014/face.evoLVe.PyTorch#94]
I want to ask how to create a file like lfw or ageDB like data zoo from original data.

**Comment by ts0526** [comment]
> I want to ask how to create a file like lfw or ageDB like data zoo from original data.

Yes, I also wonder if I can transform my own datasets to this format or structure.
